components/serializer.rst

```diff
@@ -1108,8 +1108,10 @@ always as a collection.
     behavior can be changed with the optional context key ``XmlEncoder::DECODER_IGNORED_NODE_TYPES``.
 
     Data with ``#comment`` keys are encoded to XML comments by default. This can be
-    changed with the optional ``$encoderIgnoredNodeTypes`` argument of the
-    ``XmlEncoder`` class constructor.
+    changed by adding the ``\XML_COMMENT_NODE`` option to the ``XmlEncoder::ENCODER_IGNORED_NODE_TYPES`` key of the ``$defaultContext``  of the
```

Comment:
We use to break lines at 80 characters.

Reply:
thank you @HeahDude for these corrections. It is ok for me